lib/engines/content_block_manager/app/models/content_block_manager/content_block/document.rb

```diff
@@ -27,7 +27,11 @@ class Document < ApplicationRecord
       scope :live, -> { where.not(latest_edition_id: nil) }
 
       def embed_code
-        "{{embed:content_block_#{block_type}:#{content_id}}}"
+        "#{embed_code_prefix}}}"
+      end
+
+      def embed_code_for_field(field_path)
```

Comment:
I think this is OK for now, but yeah, I think we can probably do something a bit smarter given the schema etc